machine_learning/forecasting/run.py

```diff
@@ -0,0 +1,117 @@
+"""
+this is code for forecasting
+but i modified it and used it for safety checker of data
+for ex: you have a online shop and for some reason some data are
+missing (the amount of data that u expected are not supposed to be)
+        then we can use it
+*ps : 1. ofc we can use normal statistic method but in this case
+         the data is quite absurd and only a little^^
+      2. ofc u can use this and modified it for forecasting purpose
+         for the next 3 months sales or something,
+         u can just adjust it for ur own purpose
+"""
+
+import numpy as np
+import pandas as pd
+from sklearn.preprocessing import Normalizer
+from sklearn.svm import SVR
+from statsmodels.tsa.statespace.sarimax import SARIMAX
+
+
+def lin_reg_pred(train_dt, train_usr, train_mtch, test_dt, test_mtch):
+    # linear regression, return floats
+    x = []
+    for i in range(len(train_dt)):
+        x.append([1, train_dt[i], train_mtch[i]])
+    x = np.array(x)
+    y = np.array(train_usr)
+    beta = np.dot(np.dot(np.linalg.inv(np.dot(x.transpose(), x)), x.transpose()), y)
+    prediction = abs(beta[0] + test_dt[0] * beta[1] + test_mtch[0] + beta[2])
+    return prediction
+
+
+def sarimax_predictor(train_user, train_match, test_match):
+    # sarimax, return list of float
+    order = (1, 2, 1)
+    s_order = (1, 1, 0, 7)
```

Comment:
Can you please use more self documenting variable names for order and s_order?  What are these things? The variable names should help the reader to understand that.  seasonal_order Is a better variable name than s_order.

Reply:
Please add a comment that describes what SARIMAX is doing so the reader does not need to search the net for that statsmodels documentation.

Reply:
yeah but when i used seasonal order it will be more than line limit and when i used flake it will keep telling me the warning


about description of sarimax --- ok will do it